typescript/utils/src/messages.ts

```diff
@@ -67,3 +74,19 @@ export function parseMessage(message: string): ParsedMessage {
   const body = utils.hexlify(buf.slice(BODY_OFFSET));
   return { version, nonce, origin, sender, destination, recipient, body };
 }
+
+export function parseTokenMessage(messageBody: string): ParsedTokenMessage {
+  const RECIPIENT_OFFSET = 0;
+  const AMOUNT_OFFSET = 32;
+  const buf = fromHexString(messageBody);
+  const recipient = toHexString(
+    buf.slice(RECIPIENT_OFFSET, RECIPIENT_OFFSET + 32),
+  );
+  const amount = ethers.BigNumber.from(
```

Comment:
Can we avoid ethers BigNumber here? Most of the utils package code uses `bignumber.js`. You may remember a while ago we considered a few options (including native BigInt) and opted for bignumber.js because it's one of the few that supports decimals.

Reply:
switched to bigint